src/services/request.ts

```diff
@@ -23,6 +23,7 @@ export async function request (obj: RequestParameters): Promise<string> {
       if (request.status >= 200 && request.status < 300) {
         resolve(request.responseText);
       } else {
+        console.log(request.responseText);
```

Comment:
Should we remove it?

Reply:
nah, it's for when it errors so it's valuable feedback